misc/acrn-config/library/launch_cfg_lib.py

```diff
@@ -462,6 +462,10 @@ def args_aval_check(arg_list, item, avl_list):
     """
     # args should be set into launch xml from webUI
     i_cnt = 1
+    skip_check_list = ['gvt_args', 'rootfs_dev']
+    if item in skip_check_list:
+        return
```

Comment:
If these two options are needed, it also will be skipped?

Reply:
yes, the two arguments are not must for some launch vm script. 
The two arguments can only be chosen from webUI which always showing valid options. So we don't need to check these items.